cmd/lncli/arg_parse.go

```diff
@@ -0,0 +1,40 @@
+package main
```

Comment:
Can you do this code movement in the same commit as the code was introduced? :)

Reply:
I merged all commits into one. Is that ok? Only merging some commits seems arbitrary.

I am not sure about the code coverage decrease. Do you see sth that would be sensible to test?

Reply:
I also do not know what that itest failure is about. It seems to be some timeout, maybe just try again? No changes from my side.

Reply:
Yeah, test failures are unrelated :)